taxonomy-api/taxonomy-service/app/utils/ApiId.scala

```diff
@@ -6,7 +6,10 @@ object ApiId {
 	
 	final val CREATE_FRAMEWORK = "api.framework.create"
 	final val UPDATE_FRAMEWORK = "api.framework.update"
-	
+	final val RETIRE_FRAMEWORK = "api.framework.retire"
+	final val READ_FRAMEWORK = "api.framework.read"
+	final val UPDATE_FRAMEWORK_CATEGORY = "api.frameworkCategory.update"
+	final val  RETIRE_FRAMEWORK_CATEGORY = "api.frameworkCategory.retire"
```

Comment:
Update both the values. Please check the format of `create` and `read` APIs.

```suggestion
	final val UPDATE_FRAMEWORK_CATEGORY = "api.framework.category.update"
	final val  RETIRE_FRAMEWORK_CATEGORY = "api.framework.category.retire"
```